libgnucash/backend/dbi/gnc-backend-dbi.cpp

```diff
@@ -599,28 +616,29 @@ adjust_sql_options (dbi_conn connection)
 template <DbType Type> bool
 drop_database(dbi_conn conn, const UriStrings& uri)
 {
+    // The user may have CREATE privileges even if it does not have access to the "root_db"
```

Comment:
Some might find referring to humans with "it" offensive. Let's use "they" instead.